FSharp.Compiler.Tools.Nuget/build/FSharp.Compiler.Tools.props

```diff
@@ -14,20 +14,36 @@ WARNING:  You CAN MODIFY this file, doesnt matter if you are not knowledgeable a
 
   <PropertyGroup>
 
-    <!-- Default to netcoreapp fsc -->
-    <FscTaskAssemblyPath>$(MSBuildThisFileDirectory)netcoreapp1.0/FSharp.Build.dll</FscTaskAssemblyPath>
-    <FscToolPath>$(MSBuildThisFileDirectory)netcoreapp1.0</FscToolPath>
-    <FscToolExe>fsc.dll</FscToolExe>
-    <FscToolFullPath>$(MSBuildThisFileDirectory)netcoreapp1.0/fsc.dll</FscToolFullPath>
+    <!-- Note: You may need to modify your fsproj to actually use this, otherwise it may just get overwritten -->
+    <FSharpTargetsPath>$(MSBuildThisFileDirectory)../tools/Microsoft.FSharp.Targets</FSharpTargetsPath>
```

Comment:
no, please. like that is for everyone.
one thing is supprot multiple framework fsc, but that will cause issue with new fsproj